configs/config.go

```diff
@@ -7,7 +7,7 @@ import (
 
 // ConfigModel ...
 type ConfigModel struct {
-	Port, EnvMode, SegmentWriteKey string
+	Port, EnvMode, SegmentWriteKey, TrackerTopic, TrackerProject string
```

Comment:
Looking at this codebase for the first time, I would have expected some documentation here about what is a `tracker` and what it is doing. What do you think about adding a one-sentence summary here or other places, such as the README?

Reply:
`tracker` is a name I invented so it doesn't hold any special meaning. It is a good idea to document the new endpoint and the participants, also configuration. I'll update the README